docs/config-olap-databases.md

```diff
@@ -4,26 +4,38 @@ title: OLAP Database Configuration
 sidebar_label: OLAP Database
 ---
 
-## Section
+## Introduction
 
-`olap_database:` The OLAP (online analytical processing) database section configures the OLAP database that BuildBuddy uses to enable the Trends page. **Optional**
+To speed up the analysis of historical build event data, BuildBuddy relies on ClickHouse, an OLAP database solution.
```

Comment:
```suggestion
To speed up the analysis of historical build event data, BuildBuddy can be configured to use ClickHouse as an OLAP database, in addition to the primary SQL database required for core functionality.
```

Reply:
I think these suggested edits got reverted somehow

Reply:
Yeah, it's accidental. I fixed it in the latest version.

Could you please take another look?